docs/03_concepts/03_storages.mdx

```diff
@@ -38,6 +38,10 @@ Each storage is then stored in its own folder, named after the storage, or calle
 
 Each dataset item, key-value store record, or request in a request queue is then stored in its own file in the storage folder. Dataset items and request queue requests are always JSON files, and key-value store records can be any file type, based on its content type. For example, the Actor input is typically stored in `storage/key_value_stores/default/INPUT.json`.
 
+## Local Actor run with non-local storage
```

Comment:
Awesome! Could you also mention that the typical use is when you want to call another Actor and then access its default dataset/kvstore/rq?